tests/E2E/AssetTest.php

```diff
@@ -200,7 +200,7 @@ public function testUploadAsset()
         $this->assertEquals('image/svg+xml', $asset->getFile('en-US')->getContentType());
         $this->assertContains('contentful.com', $asset->getFile('en-US')->getUrl());
 
-        $upload = $manager->getUpload($upload->getSystemProperties()->getId());
+        $upload = $manager->resolveLink($upload->asLink());
```

Comment:
For the reviewer:
this is a bit cheating, I'm getting code coverage for free adapting another test. But testing uploads is annoying, so 🤷‍♂️